installer/generate_secrets.py

```diff
@@ -242,6 +243,17 @@ def _pull_secret(self):
             "pull-secret", ".dockerconfigjson", ".docker/config.json to pull images"
         )
 
+    def _butler_secret(self):
+        self.input_file(
+            "butler-secret", "aws-credentials.ini", "AWS credentials for butler"
+            )
+        self.input_file(
+            "butler-secret", "postgres-credentials.txt", "Postgres credentials for butler"
+            )
+        self.input_file(
+            "butler-secret", "butler-hmac-idf-creds.json", "Butler credentials (HMAC JSON format)"
```

Comment:
I don't quite understand it, but Hsin-Fang knows.  I don't know that we need to support it, but it's just a reformatting of the aws-credentials.ini.  From Google's perspective it's a legacy key format for external access to the S3 bucket.  I'll drop it from the secret since we don't directly reference it and we can rederive it from the aws-credentials.ini.